nautobot/extras/templatetags/plugins.py

```diff
@@ -43,8 +45,15 @@ def _get_registered_content(obj, method, template_context):
         # Call the method to render content
         instance = template_extension(context)
         content = getattr(instance, method)()
+        if isinstance(content, dict):
+            for k, v in content.items():
+                objects.append({f"{plugin_name}:{obj_namespace}:{k}": v})
```

Comment:
I get namespacing by plugin name, but why do we need `obj_namespace`? This function is already called for a specific model as it is.

Reply:
Fair point!